Firestore/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
 - [changed] Replaced the `SetOptions` object with a simple boolean. Instead of
   calling `setData(["a": "b"], options: SetOptions.merge())` call
   `setData(["a": "b"], merge: true)`.
+- [changed] Replaced the `SnapshotOptions` object with direct use of the
+  `FIRServerTimestampBehavior` on `DocumentSnapshot`. Instead of calling
+  `get(field, SnapshotOptions.serverTimestampBehavior(.estimate))` call
+  `get(field, serverTimestampBehavior: .estimate)`.
```

Comment:
I feel like `data` is the more widely used function. Without making this overly long, can we somehow include it as well?

Reply:
I've changed the main text to `data` and added a "Changed `get` similarly."